crates/napi/src/threadsafe_function.rs

```diff
@@ -211,17 +215,16 @@ impl<T: 'static, ES: ErrorStrategy::T> ThreadsafeFunction<T, ES> {
         initial_thread_count,
```

Comment:
This might've been an issue already before, but I think there is no safe way to release a function, created with `inital_thread_count > 1`. The only place where release could happens is `drop`, and even there, `release` will be skipped if `ref_count` is greater than 1. 

Not sure how to fix it though. Maybe it is worth removing ability to set `initial_thread_count` from public API altogether and always use 1 internally? You'll still need to clone `ThreadsafeFunction` to use it from multiple threads, so `initial_thread_count > 1` does not make much sense in Rust world anyway.

Reply:
Yes, `initial_thread_count` is not public, and it's always `1`